packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -1682,3 +1959,253 @@ class PrioritizedAction extends Action<PrioritizedIntents> {
     _selectedAction.invoke(_selectedIntent);
   }
 }
+
+mixin _OverridableActionMixin<T extends Intent> on Action<T> {
+  // Asserts when the override calls this action's `invoke` method when the
```

Comment:
This sentence doesn't make sense. Should the second `when` be a `then`?

Oh, or maybe I'm reading "Asserts" as "Asserts that" instead of "This will throw an assertion exception". Maybe just make sure this is a complete sentence (and the other assert flags below).

I think I'd write this first one as:

```dart
// When debugAssertMutuallyRecursive is true, the action will throw an assertion
// error when the override calls this action's "invoke" method and the override
// is already being invoked from within the "invoke" method.
```